utilities/blob_db/blob_db.h

```diff
@@ -151,6 +151,15 @@ class BlobDB : public StackableDB {
                      ColumnFamilyHandle* column_family, const Slice& key,
                      PinnableSlice* value) override = 0;
 
+  // Get value and expiration.
+  virtual Status Get(const ReadOptions& options,
```

Comment:
Why do we need new Get APIs? Why not just add a new param to the existing Get API with a default value of nullptr?

Reply:
The original `Get` is inherited from `DB`, and we need to tell the difference of them. I would like to have a different name for the new method, but I cannot come up with a good one.

Reply:
Oh, I see. That makes sense. Sorry, I missed that.